### PR TITLE
Fix console messages on Windows with Schedule:File or other extra files

### PR DESCRIPTION
**Pull request by mjwitte**
Pull request overview
---------------------
Addresses #6793. Added a unit test, but the test doesn't fail when run without this fix. So, not sure if we want to keep it there. Run 5ZoneTDV as a test on Windows to confirm that progress messages now appear as normal.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things

### CODE REVIEW

**Comment by mjwitte** [comment]
@Myoldmopar This should be ready for review. Not sure why [this failure](https://nrel.github.io/EnergyPlusBuildResults/EnergyPlus-673f073e931acb3bb91544e9ba58c7ada01f6d6d-x86_64-MacOS-10.9-clang.html) happened on Mac. I've merged in develop locally and this test runs OK.

File: src/EnergyPlus/UtilityRoutines.cc [hunk 1]
```diff
@@ -1007,6 +1007,9 @@ int FindUnitNumber(std::string const &FileName) // File name to be searched.
         std::string::size_type TestFileLength;
         std::string::size_type Pos; // Position pointer
         for (UnitNumber = 1; UnitNumber <= MaxUnitNumber; ++UnitNumber) {
+            // Skip preassigned units - gio::inquire breaks std::cout on Windows - see GlobalStreams::
+            if ((UnitNumber == 0) || (UnitNumber == 5) || (UnitNumber == 6) || (UnitNumber == 100) || (UnitNumber == 101) || (UnitNumber == 102))
+                continue;
```
Comment (jasondegraw) [inline src/EnergyPlus/UtilityRoutines.cc]: Could we enclose this line in braces so it is clear that the `continue` goes with the `if` and not the block below?

**Review by jasondegraw — CHANGES_REQUESTED** [review]
Had one small nitpick, but this fixes the issue for me on Windows.

File: src/EnergyPlus/UtilityRoutines.cc (hunk 1 above)
Comment (mjwitte) [reply src/EnergyPlus/UtilityRoutines.cc]: OK - added braces and updated comment which was incomplete before. Ready again.

**Comment by Myoldmopar** [comment]
@mjwitte that test failure happens intermittently.  It's something missing in a clear_state, but I haven't had a chance to dig through and see what isn't being cleaned up.  

I'm going to go ahead and merge this now, I don't want to waste CI cycles on it.  

@jasondegraw thanks for reviewing this!

**Pull request merged**